Specs/DateTime/English/DateParser.json

```diff
@@ -2355,4 +2355,76 @@
     }
   ]
 }
+,
+{
+  "Input": "I'll see you the first Friday of next month",
+  "Context": {
+    "ReferenceDateTime": "2018-03-20T09:58:00"
+  },
+  "NotSupported": "javascript",
```

Comment:
If you fixed it in JavaScript, please remove the NotSupported: javascript tags from the test specs.

Reply:
OK, I'll remove it.